Landing Page

joverton2697 opened the pull request:
Landing page is mostly finished, final images and text need to be decided. Header has blank space underneath

Current Site
<img width="390" alt="Screen Shot 2020-02-11 at 1 48 50 PM" src="https://user-images.githubusercontent.com/35780852/74268103-4b1a9600-4cd5-11ea-9c6c-f6a3a0cc2435.png">
<img width="449" alt="Screen Shot 2020-02-11 at 1 48 56 PM" src="https://user-images.githubusercontent.com/35780852/74268112-4e158680-4cd5-11ea-871c-00ffc3fb3126.png">

Figma
<img width="289" alt="Screen Shot 2020-02-11 at 1 49 32 PM" src="https://user-images.githubusercontent.com/35780852/74268154-5bcb0c00-4cd5-11ea-8cff-2707b329f205.png">

Commit:
Made the contact page

Commit:
Make conditional class

Commit:
Create Services Page

Commit:
get rid of id's

Haon1919 committed:
Merge branch 'develop' into contact

Commit:
Use the FormInput Component

Haon1919 committed:
Calendar html + render helper

Begin html and add a render function to construct the calendar with proper dates for the respective month.

joverton2697 committed:
Landing Page

Landing page is mostly finished, final images and text need to be decided. Header has blank space underneath

Haon1919 commented on src/pages/Landing.tsx:
```diff
@@ -1,9 +1,37 @@
+            </section>
+            <div className={styles.content}>
+                <p>text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text </p>
+                <button onClick={learnMore}>Learn More</button><br/>
```
This should be a link.

Haon1919 commented on src/pages/Landing.tsx:
```diff
@@ -1,9 +1,37 @@
+        history.push("/About");
+    }
+
+    const enrollClick = () => {
```
Same as above.

Haon1919 commented on src/pages/Landing.tsx:
```diff
@@ -1,9 +1,37 @@
+
+    let history = useHistory();
+
+    const learnMore = () => {
```
You shouldnt need this because you should be using links.

Haon1919 commented on src/pages/Landing.tsx:
```diff
@@ -1,9 +1,37 @@
+                <div className={styles.headerText}>
+                    <h1>WHITNEY TWIGG</h1>
+                    <h2>Violin, Viola, Voice Instructor</h2>
+                    <button onClick={enrollClick}>Enroll Today!</button>
```
Needs to be a link.

Haon1919 commented on src/pages/Landing.tsx:
```diff
@@ -1,9 +1,37 @@
+                <button onClick={learnMore}>Learn More</button><br/>
+                <img className={styles.picture} src={whitneyAbout}/>
+                <p>text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text </p>
+                <button onClick={enrollClick}>Enroll Now!</button>
```
Needs to be a link.

Haon1919 commented on src/pages/Landing.tsx:
```diff
@@ -1,9 +1,37 @@
     return (
         <section>
-
+            {/*TODO header goes here*/}
```
Was this left over or did you add this? It looks like the header was in your picture. We should not be adding it in this file.

joverton2697 replied:
Yeah I think that was just left over

Haon1919 commented on src/pages/Landing.tsx:
```diff
@@ -1,9 +1,37 @@
-
+            {/*TODO header goes here*/}
+            <section className={styles.background}>
+                <div className={styles.headerText}>
```
Shouldn't need to wrap this in a div since it is the only element in the section.

joverton2697 replied:
It's wrapped in a div because the section contains the background image and wrapping the text and link in the div allow me to move them within the background section

Haon1919 commented on src/pages/Landing.tsx:
```diff
@@ -1,9 +1,37 @@
+                <p>text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text </p>
+                <button onClick={learnMore}>Learn More</button><br/>
+                <img className={styles.picture} src={whitneyAbout}/>
+                <p>text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text </p>
```
Text looks a little big in the picture you sent

Haon1919 commented:
The buttons look to small too. Maybe make those bigger and use the buttons I have on the create account page for consistency.

joverton2697 committed:
Implementing feedback

joverton2697 commented:
Implemented your feedback, attaching new photos
<img width="447" alt="Screen Shot 2020-02-11 at 2 54 31 PM" src="https://user-images.githubusercontent.com/35780852/74273432-748bef80-4cde-11ea-9d8e-36bec9f25784.png">
<img width="504" alt="Screen Shot 2020-02-11 at 2 54 37 PM" src="https://user-images.githubusercontent.com/35780852/74273445-79e93a00-4cde-11ea-83e4-87743b017afc.png">

Commit:
Make changes to Contact Page

Commit:
Make styling change to services

Commit:
Change content

Commit:
Get started on Blog Page

Commit:
Finish up Blog Component

Commit:
get Started on Forgot Password Page

Commit:
Finish Forgot Password Page

Haon1919 committed:
Got the repository battle ready

Haon1919 committed:
Got the repository battle ready

Finish calendar, add schedule lesson and lesson page, fix header z-index issue, add fake login functionality.

Haon1919 committed:
Merge remote-tracking branch 'origin/Calendar' into Calendar

Haon1919 committed:
Merge branch 'forgot-password' into Demov1

Haon1919 committed:
Merge remote-tracking branch 'origin/blog' into Demov1

Haon1919 committed:
Merge remote-tracking branch 'origin/Services' into Demov1

Haon1919 committed:
Merge remote-tracking branch 'origin/contact' into Demov1

Haon1919 committed:
Merge branch 'Demov1' into Landing

Haon1919 committed:
Merge remote-tracking branch 'origin/blog' into PresentationBranch

Haon1919 committed:
Merge branch 'forgot-password' into PresentationBranch

Haon1919 committed:
Merge remote-tracking branch 'origin/Services' into PresentationBranch

Haon1919 committed:
Merge remote-tracking branch 'origin/contact' into PresentationBranch

Haon1919 committed:
Merge remote-tracking branch 'origin/Landing' into PresentationBranch

Haon1919 committed:
Fix Login link in header

Fix login link in header by prefacing it with a slash

Commit:
Fix typo and mini bug

Haon1919 committed:
General tab remains collapsed when user logs out

Fix bug where if the user logs out and the Students tab is open, the general tab remains collapsed.

Haon1919 committed:
Merge remote-tracking branch 'origin/PresentationBranch' into PresentationBranch

Haon1919 committed:
Nonrequired form blocking submit if fails validation

If a non required field fails validation, make sure that when they remove the text it removes the failing field from the list of failing fields.

Haon1919 committed:
Minor form failing error persists for non minors

Fix error where when the birth date of the user made them a minor,and the user entered an invalid email for the guardian email then switched the birth date to make them an adult and filled in all of the other forms correctly, the  invalid guardian email still disallowed them from submitting even though they guardian email is not an option for adults.

Haon1919 committed:
Merge branch 'PresentationBranch' into Landing
